### PR TITLE
[bugfix](inverted index) fix inverted index to support NULL value filter

### PR DESCRIPTION
**Pull request by xiaokang**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by xiaokang** [comment]
run buildall

**Comment by xiaokang** [comment]
run buildall

**Comment by xiaokang** [comment]
run p0

**Comment by xiaokang** [comment]
run buildall

**Comment by xiaokang** [comment]
run buildall

File: regression-test/suites/inverted_index_p0/test_inverted_index_null.groovy [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    // test in predicates
+    qt_select_in_1 "SELECT * FROM ${table1} WHERE city IN ('shanghai', 'beijing') ORDER BY id"
+    qt_select_in_2 "SELECT * FROM ${table1} WHERE city NOT IN ('shanghai', 'beijing') ORDER BY id"
+    qt_select_in_3 "SELECT * FROM ${table1} WHERE n IN (100, 300) ORDER BY id"
+    qt_select_in_4 "SELECT * FROM ${table1} WHERE n NOT IN (100, 300) ORDER BY id"
+
+    // test match predicates
+    qt_select_match_1 "SELECT * FROM ${table1} WHERE addr MATCH_ANY 'addr fengtai' ORDER BY id"
+    qt_select_match_2 "SELECT * FROM ${table1} WHERE addr MATCH_ALL 'addr fengtai' ORDER BY id"
+}
```
Comment (Tanya-W) [inline regression-test/suites/inverted_index_p0/test_inverted_index_null.groovy:80]: this pr also can add some is_null or is_not_null predicates case, but need change the function `SegmentIterator::_inverted_index_not_support_pred_type` in segment_iterator.cpp

Comment (xiaokang) [reply regression-test/suites/inverted_index_p0/test_inverted_index_null.groovy:88]: added

**Review by Tanya-W — DISMISSED** [review]
LGTM

**Comment by xiaokang** [comment]
run buildall

**Review by qidaye — APPROVED** [review]
LGTM

**Pull request merged**